server/routes/termdb.DE.ts

```diff
@@ -98,8 +98,8 @@ param{}
 	const cases_string = group1names.map(i => i).join(',')
 	const controls_string = group2names.map(i => i).join(',')
 	const expression_input = {
-		case: cases_string,
-		control: controls_string,
+		case: controls_string,
```

Comment:
why swapped?

Reply:
@karishma-gangwani wants the 1st group to be control and 2nd group to be treatment.

Reply:
later on we can let user choose. for now default control group is 1.